main.js

```diff
@@ -0,0 +1,86 @@
+let turn = "X"
+let turnCounter = 0;
+
+const cells = Array.from(document.getElementsByClassName("cell"))
+const board = document.getElementById("board")
+const clearBtn = document.getElementById("clear-btn")
+const turnHeader = document.getElementById("turn-header")
+
+function boardClicked(event) {
+    const cell = event.target
+
+    console.log(cell.id + " clicked")
+
+    if (cell.className != "cell" || cell.innerText !== "") {
+        return
+    }
+
+    cell.innerText = turn
+    turnCounter++;
+
+    const winner = checkWinner()
+
+    if (!winner && turnCounter < 9) {
+        turn = turn == "X" ? "O" : "X";
+        turnHeader.innerText = `It's ${turn} turn`
+    }
+    else if (!!winner) {
+        turnHeader.innerText = `The winner is ${winner}!`
+        board.removeEventListener("click", boardClicked)
```

Comment:
Instead of removing and readding the listener, you can also just add an if statement iat the start of the handler